apps/settings/js/panels/wifi/wifi_wps.js

```diff
@@ -72,7 +71,9 @@ define(function(require) {
 
         req.onsuccess = function() {
           if (method === 'myPin') {
-            alert(_('wpsPinInput', { pin: req.result }));
+            navigator.mozL10n.formatValue('wpsPinInput', {
+              pin: req.result
+            }).then(msg => alert(msg));
```

Comment:
here it makes sense to separate codepaths - if not myPin then set `inProgress = true;` and fire `onSuccess();`, otherwise do those two in a callback to formatValue after alert.

Does it sound good to you @evelynhung ?